tests/data/stac/issue_copernicus_global_mosaics/stac.dataspace.copernicus.eu/v1/search_limit=20&bbox=2.1_35.31_2.2_35.32&datetime=2023-01-01t00_00_00z%2f2023-01-01t23_59_59.999000z&collections=sentinel-2-global-mosaics.json

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
`tests/data/stac/issue_copernicus_global_mosaics/stac.dataspace.copernicus.eu/v1/search_limit=20&bbox=2.1_35.31_2.2_35.32&datetime=2023-01-01t00_00_00z%2f2023-01-01t23_59_59.999000z&collections=sentinel-2-global-mosaics.json`

that's quite a funky file name. I'd strip out all special characters, unless there is a reason you left them in

Reply:
no need, it is just the URL with problematic special characters replaced with underscores. The file is now renamed.